wave_vision/include/wave/vision/fast_detector.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef WAVE_FAST_DETECTOR_HPP
+#define WAVE_FAST_DETECTOR_HPP
+
+// Libwave Headers
+#include "wave/vision/feature_detector.hpp"
+
+namespace wave {
+
+class FASTDetector : public FeatureDetector<cv::Mat> {
+ public:
+    // Public Functions
+    // -------------------------------------------------------------------------
+
+    FASTDetector(const std::string& config_path);
+    ~FASTDetector();
+    void configure(int  new_threshold,
+                   bool new_nonmax_suppression,
+                   int  new_type);
+    void getConfiguration(int&  current_threshold,
```

Comment:
changed, please take a look at this new implementation. I removed the configuration member variables, as it seemed no longer necessary.